modules/test-jpa/src/main/java/com/devonfw/module/test/common/base/clean/TestCleanerPluginLiquibase.java

```diff
@@ -0,0 +1,48 @@
+package com.devonfw.module.test.common.base.clean;
+
+import javax.inject.Inject;
+
+import org.flywaydb.core.Flyway;
+
+import liquibase.Liquibase;
+
+/**
+ * Implementation of {@link TestCleanerPlugin} base on {@link Liquibase}. It will {@link Liquibase#dropAll() clean}
+ * on {@link #cleanup()}. Therefore after {@link #cleanup()} it will drop all database objects.
+ */
+public class TestCleanerPluginLiquibase implements TestCleanerPlugin {
+
+  @Inject
+  private Liquibase liquibase;
+
+  /**
+   * The constructor.
+   */
+  public TestCleanerPluginLiquibase() {
+
+    super();
+  }
+
+  /**
+   * The constructor.
+   *
+   * @param liquibase the {@link Liquibase} instance.
+   */
+  public TestCleanerPluginLiquibase(Liquibase liquibase) {
+
+    super();
+    this.liquibase = liquibase;
+  }
+
+  @Override
+  public void cleanup() {
+
+    try {
+      this.liquibase.dropAll();
+    } catch (Exception e) {
+      e.printStackTrace();

```

Comment:
never use `printStackTrace()` in productive code. IMHO better than logging is to rethrow as runtime exception and let the test fail instead of silently ignoring a severe error.